README-v0.5.0.md

```diff
@@ -0,0 +1,713 @@
+subsequently evolved by the community, at
+[Web Access Control Wiki](https://www.w3.org/wiki/WebAccessControl). This spec
+is a particular subset of the options and extensions described in the wiki.
+
+For use with [LDP](https://www.w3.org/TR/ldp/) (and [LDP
+Next](https://www.w3.org/community/ldpnext/)) type systems, such as the
+[Solid](https://github.com/solid/solid) project (see also the parent
+[spec](https://github.com/solid/solid-spec)).
+
+**Current Spec version:** `v.0.5.0` (see [CHANGELOG.md](CHANGELOG.md))
+
+## Table of Contents
+
+1. [Overview](#overview)
+2. [Access Control List Resources](#access-control-list-resources)
+  * [Containers and Inherited ACLs](#containers-and-inherited-acls)
+  * [Individual Resource ACLs](#individual-resource-acls)
+  * [ACL Resource Location Discovery](#acl-resource-location-discovery)
+3. [ACL Inheritance Algorithm](#acl-inheritance-algorithm)
+4. [Representation Format](#representation-format)
+5. [Example WAC Document](#example-wac-document)
+6. [Describing Agents](#describing-agents)
+  * [Singular Agent](#singular-agent)
+  * [Groups](#groups-of-agents)
+  * [Public Access (all Agents)](#public-access-all-agents)
+  * [Anyone logged on (Authenticated Agents)](#authenticated-agents-anyone-logged-on)
+  * [Referring to Origins, i.e. Web Apps](#referring-to-origins-ie-web-apps)
+7. [Referring to Resources](#referring-to-resources)
+8. [Modes of Access](#modes-of-access)
+9. [Default (Inherited) Authorizations](#default-inherited-authorizations)
+10. [Not Supported by Design](#not-supported-by-design)
+
+## Overview
+
+Web Access Control (WAC) is a decentralized cross-domain access control system.
+The main concepts should be familiar to developers, as they are similar to
+access control schemes used in many file systems. It's concerned with giving
+access to agents (users, groups and more) to perform various kinds of operations
+(read, write, append, etc) on resources. WAC has several key features:
+
+1. The resources are identified by URLs, and can refer to any web documents or
+  resources.
+2. It is *declarative* -- access control policies live in regular web documents,
+  which can be exported/backed easily, using the same mechanism as you would
+  for backing up the rest of your data.
+3. Users and groups are also identified by URLs (specifically, by
+  [WebIDs](https://github.com/solid/solid-spec#identity))
+4. It is *cross-domain* -- all of its components, such as resources, agent
+  WebIDs, and even the documents containing the access control policies, can
+  potentially reside on separate domains. In other words, you can give access
+  to a resource on one site to users and groups hosted on another site.
+
+## Access Control List Resources
+
+In a system that uses Web Access Control, each web resource has a set of
+*Authorization* statements describing:
+
+1. Who has access to that resource (that is, who the authorized *agents* are)
```

Comment:
This is true for `acl:accessTo` Authorizations. But there are three types of Authorizations:
* `acl:accessTo`
* `acl:default`
* `acl:accessTo + acl:default`

For `acl:default` and `acl:accessTo + acl:default` Authorizations, it's (additionally) possible to describe who has access to child resources who don't have their own ACL doc.

Reply:
Ah wait, now I see what you did here. OK, I was put on the wrong leg by the "has a set of" phrase. Maybe "has a set of applicable" would be clearer? That talks about which Authorizations (directly or indirectly) apply to a resource, not which Authorizations "belong" to the resource.

Reply:
Wait.. READ-v0.5.0.md is just moved from README.md. README.md is then updated to reflect current state, moving off Editor's Draft.